launch/small_house_turtlebot_navigation.launch.py

```diff
@@ -0,0 +1,103 @@
+# /*******************************************************************************
+# * Copyright 2019 ROBOTIS CO., LTD.
+# *
+# * Licensed under the Apache License, Version 2.0 (the "License");
+# * you may not use this file except in compliance with the License.
+# * You may obtain a copy of the License at
+# *
+# *     http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# *******************************************************************************/
+
+# /* Author: Darby Lim */
+
+import os
+from ament_index_python.packages import get_package_share_directory
+import launch
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.actions import IncludeLaunchDescription
+from launch.conditions import IfCondition
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import LaunchConfiguration
+
+TURTLEBOT3_MODEL = os.environ['TURTLEBOT3_MODEL']
+
+def generate_launch_description():
+    use_sim_time = launch.substitutions.LaunchConfiguration('use_sim_time', default='false')
+    world_file_name = 'small_house.world'
+    package_dir = get_package_share_directory('aws_robomaker_small_house_world')
+    gazebo_ros = get_package_share_directory('gazebo_ros')
+
+    gazebo_client = launch.actions.IncludeLaunchDescription(
```

Comment:
Could this launch file just call the other launch files, eg. `small_house.launch.py` and `turtlebot3_navigation.launch.py`?

Reply:
Yes .. done the changes.